src/LCT.PackageIdentifier/NugetDevDependencyParser.cs

```diff
@@ -0,0 +1,271 @@
+﻿using log4net;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Exceptions;
+using Microsoft.Build.Locator;
+using Microsoft.Extensions.Logging;
+using NuGet.Packaging.Core;
+using NuGet.ProjectModel;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Security.Cryptography;
+using LCT.PackageIdentifier.Model.NugetModel;
+using System.Text.Json;
+using log4net.Repository.Hierarchy;
+
+namespace LCT.PackageIdentifier
+{
+    internal class NugetDevDependencyParser
+    {
```

Comment:
Check the code complexity and make sure it is below 10

Reply:
Checked